Libraries/Components/TextInput/TextInput.js

```diff
@@ -1256,7 +1255,6 @@ const TextInput = createReactClass({
         onPress={this._onPress}
         accessible={this.props.accessible}
         accessibilityLabel={this.props.accessibilityLabel}
-        accessibilityHint={this.props.accessibilityHint} // TODO(OSS Candidate ISS#2710739)
```

Comment:
We use this prop in the macOS fork.  Please restore.   The comment can be changed to `TODO(macOS ISS#2323203)`